includes/Modules/Analytics.php

```diff
@@ -1385,8 +1420,12 @@ protected function create_analytics_site_data_request( array $args = array() ) {
 			$dimension_filter->setOperator( 'EXACT' );
 			$args['page'] = str_replace( trim( $this->context->get_reference_site_url(), '/' ), '', esc_url_raw( $args['page'] ) );
 			$dimension_filter->setExpressions( array( rawurldecode( $args['page'] ) ) );
+			$dimension_filters_clauses[] = $dimension_filter;
+		}
+
+		if ( ! empty( $dimension_filters_clauses ) ) {
 			$dimension_filter_clause = new Google_Service_AnalyticsReporting_DimensionFilterClause();
-			$dimension_filter_clause->setFilters( array( $dimension_filter ) );
+			$dimension_filter_clause->setFilters( $dimension_filters_clauses );
```

Comment:
This clause should remain independent as it is in develop now, but `setFilters` should be called with `array( $dimension_filter )` like it was before rather than the array of clauses.

Reply:
That's right, changed it back.